Source/WebCore/rendering/ImageQualityController.cpp

```diff
@@ -113,7 +113,7 @@ std::optional<InterpolationQuality> ImageQualityController::interpolationQuality
     return std::nullopt;
 }
 
-InterpolationQuality ImageQualityController::chooseInterpolationQuality(GraphicsContext& context, RenderBoxModelObject* object, Image& image, const void* layer, const LayoutSize& size)
+InterpolationQuality ImageQualityController::chooseInterpolationQuality(GraphicsContext& context, RenderBoxModelObject* object, Image& image, const void* layer, const LayoutSize& layoutSize)
 {
     // If the image is not a bitmap image, then none of this is relevant and we just paint at high quality.
     if (!(image.isBitmapImage() || image.isPDFDocumentImage()) || context.paintingDisabled())
```

Comment:
I think the image interpolation quality has nothing to do with the PDFDocumentImage. So I think we either need to remove the condition `image.isPDFDocumentImage()` or we fix the comment.

Reply:
I am not modifying this in patch but if you want me to change this, I can do it as part of this patch.

Reply:
Yes. Please remove `image.isPDFDocumentImage()` from this if-statement since `PDFDocumentImage::draw()` does not call the `ImagePaintingOptions::interpolationQuality()`.

Reply:
Done - let's see.